frontend/src/app/components/modal/modal.component.html

```diff
@@ -0,0 +1,6 @@
+
```

Comment:
Using some advanced feature of angular to manage the dynamic content of a modal.

My goal was to create a modal component like this

`<modal>
whatever html or component but the modal itself dont care or dont have to handle it
</modal>`